src/main/java/org/isf/hospital/gui/HospitalBrowser.java

```diff
@@ -271,57 +271,44 @@ public JPanel getJButtonPanel() {
 			ExitJButton.setMnemonic(KeyEvent.VK_C);
 			UpdateJButton.setEnabled(false);
 			
-			ExitJButton.addActionListener(new ActionListener() {
-				public void actionPerformed(ActionEvent arg0) {
```

Comment:
removing this code means the imports for `ActionEvent` and `ActionListener` are no longer needed